client/dfclient/src/resources.cpp

```diff
@@ -8,3 +8,16 @@ void Resources::UpdateTweens()
 			_tweens.erase(_tweens.begin() + i);
 	}
 }
+
+void Resources::playKillSound(float gain) {
+	_killSound->setGain(gain);
+	_killSound->play();
+}
+
+void Resources::playRandomDeathSound() {
+	auto randIndex = rand()%_sounds.size();
+	auto it = _sounds.begin();
+	for(auto i=0; i<randIndex; ++i, ++it);
```

Comment:
the wasm build fails on this line with some error regarding instantiating a template. the native build works even though this doesn't look right to me

Reply:
Fixed by adding `#include <string>` to [resources.hpp](client/dfclient/include/resources.hpp). Still, took `++it` out of the braces for readability.